frontend/ui/widget/imagewidget.lua

```diff
@@ -57,6 +57,7 @@ local ImageWidget = Widget:new{
     -- size vertically and horizontally, without impact original aspect ratio.
     -- But overflow part will be ignored.
     overflow = false,
+    nocache = false,
```

Comment:
To skip caching, you can set the image attribute instead of the file attribute. The image attribute expects a raw uncompressed bitmap that can be blit directly to the screen. Here is an example: https://github.com/koreader/koreader/blob/master/frontend/ui/widget/bookstatuswidget.lua#L298.

Looks like you already have the jpeg image in memory through `http.request(self.dates.image)`. To decompress a jpeg to raw bitmap, you need to use the API from the jpeg library shipped with koreader. You can add a helper funciton in `ffi/pic.lua` to just decompress a jpeg in memory to a raw bitmap and use it here. If you look at the source code of `ffi/pic.lua` where it renders a jpeg file, all it does is just reading that file into memory and hand it to libturbojpeg: https://github.com/koreader/koreader-base/blob/master/ffi/pic.lua#L191. So you can reuse the same code there. If you get stuck at this, please let me know. I can help point you to the right direction or write the helper function for you if needed.

By doing everything in memory, you can skip the cache and avoid writing the image data to disk.